pkg/scanners/quay/quay.go

```diff
@@ -58,7 +58,7 @@ func newScanner(protoImageIntegration *storage.ImageIntegration) (*quay, error)
 	}
 	config := quayConfig.Quay
 
-	registry, err := quayRegistry.NewRegistryFromConfig(quayConfig.Quay, protoImageIntegration)
+	registry, err := quayRegistry.NewRegistryFromConfig(quayConfig.Quay, protoImageIntegration, false)
```

Comment:
how come this is hardcoded `false`?

Reply:
My intent was to add the capability to disable the call and then make use of it for the on-prem scanning workflows, (which were not matching on the repo list).

The Central scanning workflows (including this one) I kept as is to avoid adverse affects if users are still benefiting from it. Believe it makes more sense to disable in the future for Central scanning workflows when/if we determine is no benefit or have some other mechanism to more efficiently match an image to a registry integration.